.vscode/settings.json

```diff
@@ -5,7 +5,7 @@
     "**/node_modules": true,
     "**/build": true
   },
-  "editor.formatOnSave": true,
+  "eslint.autoFixOnSave": true,
```

Comment:
Thanks to this you can safely save the `package.json` without reformatting, you're welcome!

Reply:
would be nice to autofix md files though. Would be better to add a `.prettierignore` with package.json in it

Reply:
oh, because we don't use eslint directly to format md files? :(

Reply:
correct, it's just prettier